src/domain/individualCustomer/billsCard/BillsCard.tsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import Card from '../../../common/card/Card';
+import Text from '../../../common/text/Text';
+import Button from '../../../common/button/Button';
+import LabelValuePair from '../../../common/labelValuePair/LabelValuePair';
+import Paragraph from '../../../common/paragraph/Paragraph';
+import styles from './billsCard.module.scss';
+
+interface Props {
```

Comment:
Can we change props naming convention from `interface Props` to `export interface ComponentProps` for compatibility with HDS?